src/main/java/mate/academy/Main.java

```diff
@@ -6,16 +6,20 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
+            logger.error("Failed login attempt. Reason: {}", e.getMessage());
+            throw new RuntimeException("Username or password are incorrect" + e);
```

Comment:
Instead of concatenating the exception `e` directly to the string in the `RuntimeException`, consider using a logger method to log the exception details. This will make the log message more informative and align with the task requirements to replace `e.printStackTrace()` with logger methods.